.github/workflows/lynx.yml

```diff
@@ -0,0 +1,52 @@
+name: '🔎 Lynx Testnet Example'
+
+on:
+  schedule:
+    - cron: "0/20 * * * *"  # Every 20 minutes
+  pull_request:
+    # branches:
+    #   - main
+    #   - development
+    #   - epic-*
+  workflow_dispatch:
+
+env:  # TODO: Use variables when GH supports it for forks. See https://github.com/orgs/community/discussions/44322
+  RPC_PROVIDER_URL: "https://polygon-mumbai.infura.io/v3/3747007a284045d483c342fb39889a30"
+  PRIVATE_KEY: "0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86"
```

Comment:
Are we sure we want to expose this infura endpoint and this private key?

If someone take this private key and drain all the founds, this workflow will not pass, right?

Also, if someone drain the free queries of the infura endpoint, also this will fail.

Maybe I'm missing something.

Reply:
As far as I know, there is an option to use secrets in GitHub:

https://docs.github.com/en/actions/using-workflows/sharing-workflows-secrets-and-runners-with-your-organization#sharing-secrets-and-variables-within-an-organization

Reply:
Re: Infura endpoint, it is configured with allowlist for Taco contracts and certain JSON-RPC methods. We could even consider adding rate limits and origin restrictions, if necessary.

Re: private key, this is a test private key that's already part of the nucypher codebase.

Re: secrets in GH, the problem is that these are not accessible by pull requests triggered from forks, which is not convenient for us. That's even the case for non-secret GH variables! See for a recent support discussion in GH: https://github.com/orgs/community/discussions/44322
In any case, these particular private key and infura endpoint can be published because of the reasons mentioned earlier, so for the moment it's OK.

Reply:
I'm fine with the restricted Infura endpoint. Another option you could consider is a public endpoint for mumbai...e.g. https://rpc-mumbai.polygon.technology (from https://wiki.polygon.technology/docs/pos/reference/rpc-endpoints/)

Reply:
Ok, I still think that someone draining the wallet can cause some trouble, but we can proceed with this and see if this "attack" happens or not.